### PR TITLE
Home

### PR DESCRIPTION
**Pull request by jk0ehn**
My first pull request

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -1,12 +1,7 @@
...
 	return (
 		<header>
-			<nav>
-				<Link href="/">Home</Link>
-				<Link href="/about">About me</Link>
-			</nav>
+			<nav></nav>
 		</header>
 	);
 }
```
Comment (mbosselmann) [inline src/components/Header.js]: As you don't seem to need this component for now I would recommend you to delete it. :)

File: src/pages/[experts]/[...params].js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import {useRouter} from 'next/router';
+
+export default function ID() {
+	const router = useRouter();
+	const {params = []} = router.query;
+
+	if (params.length === 2) {
+		return (
+			<div>
```
Comment (mbosselmann) [inline src/pages/[experts]/[...params].js]: I think you don't need a `div` here as you have only one element nested inside your `div` element.

File: src/pages/[experts]/[...params].js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+				<h1>
+					Alle {params[0]} für unter {params[1]} Euro
+				</h1>
+			</div>
+		);
+	} else if (params.length === 1) {
+		return <h1>Alle {params[0]}</h1>;
+	}
+	return <h1>Autos</h1>;
```
Comment (mbosselmann) [inline src/pages/[experts]/[...params].js]: Why do you return the text "Autos"? 😅

File: src/pages/[experts]/index.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+export default function Cities() {
+	return (
+		<div>
+			<h1>City</h1>
+		</div>
+	);
```
Comment (mbosselmann) [inline src/pages/[experts]/index.js]: Same here what I described above about one nested element.

File: src/pages/index.js [hunk 5]
```diff
@@ -1,15 +1,24 @@
...
 			</Head>
-			<h1>Home</h1>
-		</Layout>
+			<div>
+				<h1>Find your local expert for medical cannabis</h1>
+				<Image src={logo} alt={logo} width="100px" height="100px" />
+				<h2>Where do you want to find a cannabis expert?</h2>
+				<Link href="/search">
+					<button>enter your search here</button>
+				</Link>
```
Comment (mbosselmann) [inline src/pages/index.js]: We don't nest a button element inside a link element. It is either a button or a link, but not both. But you can style your Link like a button if you like. [See the Next.JS Documentation ](https://nextjs.org/docs/api-reference/next/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag)and the following example for better understanding (As far as I know, you can't style a Next.JS Link component as styled component. Therefore, you need to use another anchor element):

```js
<Link href="/test" passHref>
	<StyledLink>Link</StyledLink>
</Link>

const StyledLink = styled.a`
	color: red;
`;
```

File: src/pages/index.js [hunk 6]
```diff
@@ -1,15 +1,24 @@
...
 			<Head>
-				<title key="title">My Project</title>
+				<title key="title">My Cannabis Guide</title>
 				<meta key="description" name="description" content="This is my project" />
 			</Head>
-			<h1>Home</h1>
-		</Layout>
+			<div>
+				<h1>Find your local expert for medical cannabis</h1>
+				<Image src={logo} alt={logo} width="100px" height="100px" />
```
Comment (mbosselmann) [inline src/pages/index.js]: You `alt` text should not the name of your file ("/logo.png"). It should be a meaningful name that describes what the user sees. I think the name of your app would be the best description for this image.

File: src/pages/search.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+export default function Search() {
+	return (
+		<>
+			<Head>
+				<title key="title">Search</title>
+				<meta key="description" name="description" content="About" />
+			</Head>
+			<h1>Search</h1>
+			<input placeholder="Where do you want to search"></input>
+		</>
```
Comment (mbosselmann) [inline src/pages/search.js:12]: Your input element is missing a label. A placeholder text is not a suitable substitute for a label element and should always be consider as extra. There exists also the recommendation to not use placeholder texts as their styling often results in text that is harder to read. But that is up to you to decide. 

Please check our coding examples about how to write accessible forms. [Maybe this guide is also helpful for you.](https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_structure_a_web_form)
You should also wrap your input element in a form element as every search is also a form.

**Review by mbosselmann — COMMENTED** [review]
Hi Julian,
good job! Congrats on your first pull request! 🎉 I left some remarks for you to consider before moving further. And I have one question: As your search functionality is not implemented yet the dynamic routes can't be accessed yet or? 🤔 If this is correct, I would recommend you for the future to create these components when they are needed and not in advance. :)

File: src/pages/index.js (hunk 6 above)
Comment (jk0ehn) [reply src/pages/index.js]: Its a placeholder at the moment

File: src/pages/search.js (hunk 7 above)
Comment (jk0ehn) [reply src/pages/search.js:12]: This is the next step for my coming userstory. It was only to test the link for me....

File: src/pages/index.js (hunk 6 above)
Comment (mbosselmann) [reply src/pages/index.js]: Ok :)

File: src/pages/search.js (hunk 7 above)
Comment (mbosselmann) [reply src/pages/search.js:12]: Ok! 👍

File: src/components/Link/StyledLink.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+
+const StyledLink = styled.div`
+	width: 500px;
+	max-width: 600px;
+	padding: 15px;
+	gap: 15px;
+	border-radius: 9999px;
+	color: grey;
+	box-shadow: 0px 0px 6px 1px rgba(204, 204, 204, 0.48);
+`;
```
Comment (mbosselmann) [inline src/components/Link/StyledLink.js:11]: You got one code quality remark:

```
 src/components/Link/StyledLink.js
 10:2   ✖  Expected "box-shadow" to come before "color" in group "background"   order/properties-order
 10:15  ✖  Unexpected unit                                                      length-zero-no-unit   
 10:19  ✖  Unexpected unit                                                      length-zero-no-unit
```

Just change the order of the CSS properties according to the code quality remarks and remove the px units for 0 units. :)

And your styled component should be based on an anchor element. Just check my comment. :D

**Review by mbosselmann — APPROVED** [review]
Just one remark, everything else looks fine. :+1 I approve your PR, but please fix this last remark before moving further. :)

**Pull request merged**